.github/workflows/vcpkg.yml

```diff
@@ -132,6 +135,12 @@ jobs:
         run: |
           export CL=-openmp
 
+          # `${{ matrix.cxxflags }}` is due an error in linux:
```

Comment:
Are you saying there’s an error in the compiler? Or an error in our code?

If the latter, did you try fixing it?

Reply:
You have warning on your code, and you do 2 things:
- Treat the warning as errors:
 https://github.com/borglab/gtsam/blob/develop/cmake/GtsamBuildTypes.cmake#L134
- You ignore all warning in other CI:
 https://github.com/borglab/gtsam/blob/develop/.github/scripts/unix.sh#L39
 
 I didn't try to fix the warning. I think it just remove the redundant move. 
 I can try that after the ci will be green.